From: mchristi@redhat.com <mchristi@redhat.com>
Date: Wed, 20 May 2009 15:25:18 -0500
Subject: [scsi] fnic: compile on x86 too
Message-id: 1242851118-5743-1-git-send-email-mchristi@redhat.com
O-Subject: [PATCH] RHEL 5.4: build fnic on x86 and x86_64
Bugzilla: 501112

From: Mike Christie <mchristi@redhat.com>

This is for BZ 501112.

This builds fnic on x86 and x86_64. We were only building it on
x86_64. This change is upstream.

Here is the brew build:
http://brewweb.devel.redhat.com/brew/taskinfo?taskID=1807653

diff --git a/drivers/scsi/Kconfig b/drivers/scsi/Kconfig
index d5d3498..bd4fb4c 100644
--- a/drivers/scsi/Kconfig
+++ b/drivers/scsi/Kconfig
@@ -586,7 +586,7 @@ config SCSI_OMIT_FLASHPOINT
 
 config FCOE_FNIC
 	tristate "Cisco FNIC Driver"
-	depends PCI && X86_64
+	depends PCI && X86
 	select LIBFC
 	help
 	  This is support for the Cisco PCI-Express FCoE HBA.
